src/main/resources/custom-events.sk

```diff
@@ -0,0 +1,36 @@
+
+# This file allows you to register custom events that aren't already registered under Skript.
+# The event classes you want to register must exist in the server class loader. So events from Paper/Spigot.
+# If you want to register an event from a plugin API, you must edit the plugin.yml in the jar of Skript to include:
+#
+# softdepend: [YourPlugin, SQLibrary, Vault, WorldGuard, Residence, PreciousStones, GriefPrevention]
+#
+# Reminder to maintain any existing defined plugins in this array as this example should not be directly copied.
+#! Additions or changes to this file requires a server restart.
+
+#! Internal version of this configuration. Don't modify.
+version: 1
+
+# The format must include a section where the section name is the event name.
+# The event section must include:
+# 	description - describing the event
```

Comment:
This claims you **must** include description, but it's optional and neither example has a description. You should change this to say it's optional and it has a default value, and put an example of a custom description on one of the example events.